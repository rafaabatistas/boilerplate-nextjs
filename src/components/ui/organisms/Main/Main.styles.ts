import styled from 'styled-components';

export const Wrapper = styled.main`
  width: 100vw;
  height: calc(100vh - 80px);
  background-color: #19062b;
  text-align: center;
  color: white;
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
  padding: 20px;
`;

export const Description = styled.h2`
  font-size: 1.6rem;
  margin-bottom: 2rem;
`;

export const Ilustration = styled.img`
  width: 260px;
`;
