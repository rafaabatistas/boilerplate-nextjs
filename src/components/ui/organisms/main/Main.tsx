const Main = () => <h1>React</h1>;

export default Main;
