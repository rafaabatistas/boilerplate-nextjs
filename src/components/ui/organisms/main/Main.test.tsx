import { render, screen } from '@testing-library/react';

import Main from './Main';

describe('<Main />', () => {
  it('Deve renderizar o componente <Main />', () => {
    const { container } = render(<Main />);
    const Heading = screen.getByRole('heading', { name: 'React' });

    expect(Heading).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
