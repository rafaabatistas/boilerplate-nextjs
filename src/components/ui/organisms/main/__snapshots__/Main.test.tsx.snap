// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Deve renderizar o componente <Main /> 1`] = `
<h1>
  React
</h1>
`;
